applications/eos-cryptography-proposal.md

```diff
@@ -0,0 +1,199 @@
+
+- High severity [bug discovery in EOSIO node](https://b1.com/press/eosio-ram-resource-exploit-patch/) and disclosed through B1 hacker one bounty.
+- EOS smart contract for BPs rotation.
+- EOS smart contract for RAM resource market (RAM token).
+- Providing bug fixes and patches for EOSIO CDT and EOSIO node software.
+- 3rd place with [ROW project](https://github.com/ZeroPass/row.contract) at B1 hackathon / Google Cloud Platform, for proving WebAuthn to be fully done on-chain.
+- [eosio.ck](https://github.com/ZeroPass/eosio.ck) - RSA & SHA-3 cryptography library for EOSIO smart contract.
+
+### Team Code Repos
+
+- https://github.com/ZeroPass
+- https://github.com/ZeroPass/eosio.ck
+- https://github.com/ZeroPass/eosio-port
+
+ ### Team Members GitHub Accounts
+
+- https://github.com/smlu
+- https://github.com/nejc-skerjanc
+- https://github.com/lukapercic
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/nejcskerjanc/
+- https://www.linkedin.com/in/crt-vavros/
+- https://www.linkedin.com/in/lukapercic/
+
+## Development Status
+
+**RSA PoC**
+We painstakingly built, optimized, tested and deployed on the testnet to make it the fastest wasm RSA implementation in eosio. For example, it can verify RSA 4096 bit keys consistently under 10ms (3ms on average, much faster for the more standard  RSA 2048 bit).
+- https://github.com/ZeroPass/eosio.ck
+
+
+We also built and deployed port itself (server solution) that acts as our beta version for what we are trying to move on chain. In addition, we already experimented and built Port PoC smart contract for passport attestation on-chain (RSA PKCS 1.5 PKI only). 
+- https://github.com/ZeroPass/eosio-port
+
+We can also attach research from our documentation (which is not completly up to date). It is presented in the [In-depth section](https://github.com/ZeroPass/Port-documentation-and-tools#in-depth).
+
+## Development Roadmap
+
+- **Total Estimated Duration:** 5,5-7 months
+- **Full-Time Equivalent (FTE):**  16,5-21 FTE months
+- **Total Costs:** $120.000
+
+### Milestone 1 - Opensource RSA
+
+- **Estimated duration:** 0 month
+- **FTE:** 0
+- **Costs:** 20.000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation | Documentation and step by step guide already provided in [README.md](https://github.com/ZeroPass/eosio.ck/blob/master/README.md).  | 
+| 0c. | Testing Guide | [In the guide](https://github.com/ZeroPass/eosio.ck/blob/master/README.md#algorithm-testing=), we are describing how to run those tests. |
+| 0d. | Running it | We deployed on the [Jungle 3 and CryptoKylin testnets](https://github.com/ZeroPass/eosio.ck/blob/master/README.md#testnet=). |
+| 1. | EOSIO SDK library | Open-sourcing RSA PKCS v1.5 signature verification algorithm and the Keccak hash algorithms: SHA3-256, SHA3-512, SHAKE-128 and SHAKE-256 found in the [EOSIO Cryptography Kits](https://github.com/ZeroPass/eosio.ck) |
```

Comment:
For clarification, currently this has been published without a license, correct? 

Do you have agreements with all current copywrite holders (aka contributors) to relicense with MIT?

Reply:
Yes, it's our original work, code available, but with no license provided yet (not open-sourced). The only borrowed code is [u-boot](https://github.com/u-boot/u-boot), which is GPL-2.0+. That license stays, regardless if our code is MIT (separate files).